Exiled.Events/EventArgs/Scp106/ServerChangingStalkEventArgs.cs

```diff
@@ -0,0 +1,113 @@
+// -----------------------------------------------------------------------
+// <copyright file="ServerChangingStalk.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp106
+{
+    using API.Features;
+    using Interfaces;
+    using Mirror;
+    using PlayerRoles.PlayableScps.Scp106;
+    using PlayerRoles.PlayableScps.Subroutines;
+    using UnityEngine;
+
+    /// <summary>
+    ///     Contains all information before SCP-106 server changes stalk.
+    /// </summary>
+    public class ServerChangingStalkEventArgs : IPlayerEvent, IDeniableEvent
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServerChangingStalkEventArgs"/> class.
+        /// </summary>
+        /// <param name="player"><inheritdoc cref="ServerChangingStalkEventArgs.Player" /></param>
+        /// <param name="stalkAbilityInstance"><inheritdoc cref="ServerChangingStalkEventArgs.Scp106StalkAbility" /></param>
+        /// <param name="cooldownAbility"><inheritdoc cref="ServerChangingStalkEventArgs.CooldownAbility" /></param>
+        /// <param name="isActive"><inheritdoc cref="ServerChangingStalkEventArgs.IsActive" /></param>
+        /// <param name="isAllowed"> <inheritdoc cref="ServerChangingStalkEventArgs.IsAllowed" /></param>
+        public ServerChangingStalkEventArgs(Player player, Scp106StalkAbility stalkAbilityInstance, AbilityCooldown cooldownAbility, bool isActive, bool isAllowed = true)
+        {
+            Player = player;
+            Scp106StalkAbility = stalkAbilityInstance;
+            CooldownAbility = cooldownAbility;
+            IsActive = isActive;
+            Vigor = Scp106StalkAbility.Vigor.VigorAmount;
+            Cooldown = 20f;
```

Comment:
Hard coded

Reply:
@louis1706 LMAO I forgot this existed, I'll take a look later, if I forget, ping me